src/main/resources/site/parts/employee/employee.jsx

```diff
@@ -1,109 +1,195 @@
-import React from 'react'
-import { Button, Title, Link, LeadParagraph, Paragraph, Text } from '@statisticsnorway/ssb-component-library'
-import PropTypes from 'prop-types'
-import { Share2, Send, Smartphone, Eye, Download, ExternalLink } from 'react-feather'
+import React from 'react';
+import { Button, Title, Link, LeadParagraph, Paragraph, Text, Accordion } from '@statisticsnorway/ssb-component-library';
+import PropTypes from 'prop-types';
+import { Share2, Send, Smartphone, Eye, Download, Image } from 'react-feather';
 
 const Employee = (props) => {
   const {
-    title, email, position, phone, description, profileImages, myCV, projects, isResearcher, cristinId, area,
+    title, email, position, phone, description, profileImages, myCV, projects, isResearcher, cristinId, area, cvInformation,
     emailPhrase,
     phonePhrase,
     positionPhrase,
     researchAreaPhrase,
     departmentPhrase,
     briefSummaryPhrase,
-    projectsPhrase
-  } = props
+    projectsPhrase,
+    downloadPdfPhrase, 
+    publicationsPhrase,
+    pressPicturesPhrase,
+    pressPicturesDescrPhrase, 
+    imagePhrase,
+  } = props;
+
+  const calculateCvSize = (bytes) => {
+    return Math.round(bytes / 1000)
+  }
+
+  const renderPortraitImages = () => {
+    return (
+      <div className="grid-row">
+        {profileImages.map((href, i) => {
+          return (
+            <div key={i} className="grid-column" role="img" aria-label={`${pressPicturesPhrase} ${i + 1} av ${title}`}>
+              <a href={href} target="_blank" type="media_type">
+                <div><img alt={`${pressPicturesPhrase} ${i + 1} av ${title}.`} src={href}/></div>
+                <div><Link linkType="profiled">{imagePhrase} {i + 1}.jpg</Link></div>
+              </a>
+            </div>
+          )
+        })}
+      </div>
+    )
+  }
+
+  const downloadPDF = (url) => {
+    const link = document.createElement("a");
+    link.href = url;
+    link.click();
+  }
+
+  const renderDownloadCvButton = () => {
+    return (
+      myCV ? 
+        <div className="downloadCv">
+          <Button onClick={() => downloadPDF(myCV)}><Download size="18" /> &nbsp; {downloadPdfPhrase} ({calculateCvSize(cvInformation.size)} kB) </Button>
```

Comment:
For mobile resolution; there is a little bit of a margin (.2rem) around the button.